apps/protocol-api/src/main.ts

```diff
@@ -83,6 +83,7 @@ const permissions = shield(
       linearUsers: hasToken,
       linearIssues: hasToken,
       contributionStatuses: hasToken,
+      getAverageActiveGuildUsers: or(isAuthenticated, hasToken),
```

Comment:
Shouldn't this be getGuildUsersAverageByActive

Reply:
`By` clauses refer to the grouping of the data. e.g. `..ByDay` queries return the noun bucketed by day.
 
do you feel this naming doesn't conform to what we laid out during the offsite?

Reply:
@alexkeating

Reply:
Ah okay that makes sense. Maybe the verb is average instead of get average? As we will have no other verb variants of average. So maybe averageActiveGuildUsers. And what would be averageActiveGuildUsers?

Reply:
`getContributionCountByActivityType` suggests a pattern that we could follow. maybe `getActiveGuildUsersAverage`? we'd be following e.g. `get...Count|Average|Max`

Reply:
Yea, I that makes sense verb is get and noun is ActiveGuildUsersAverage.